File: app/src/main/java/com/blues/gankio/old/model_article/ArticleHolder.java
package com.blues.gankio.old.model_article;

import android.view.ViewGroup;
import android.widget.TextView;

import com.blues.R;
import com.jude.easyrecyclerview.adapter.BaseViewHolder;

/**
 * User : Blues
 * Date : 2019/5/17
 * Time : 10:39
 */

public class ArticleHolder extends BaseViewHolder<Article.ArticleBean> {

    private TextView tvTitle;

    public ArticleHolder(ViewGroup parent) {
        super(parent, R.layout.item_article_mp);

        tvTitle = $(R.id.tv_title);
    }

    @Override
    public void setData(Article.ArticleBean data) {
        super.setData(data);

        tvTitle.setText(data.getTitle());

    }
}

File: app/src/main/java/com/blues/gankio/v1/view/ArticleHolder.kt
package com.blues.gankio.v1.view

import android.view.ViewGroup
import com.blues.gankio.v1.model.ArticleBean.ArticleDetailBean
import com.blues.R
import android.widget.TextView
import com.jude.easyrecyclerview.adapter.BaseViewHolder

/**
 * User : Blues
 * Date : 2019/5/17
 * Time : 10:39
 */
class ArticleHolder(parent: ViewGroup?) :
    BaseViewHolder<ArticleDetailBean>(parent, R.layout.item_article_mp) {

    private val tvTitle: TextView = itemView.findViewById(R.id.tv_title)

    override fun setData(data: ArticleDetailBean) {
        super.setData(data)
        tvTitle.text = data.title
    }
}